<?xml version="1.0"?>
<launch>
  <node pkg="kurt3d" type="kurt3d_teleop_keyboard" name="kurt3d_teleop" output="screen">
    <param name="walk_vel" value="0.15" />
    <param name="run_vel" value="0.3" />
    <param name="yaw_rate" value="0.3" />
    <param name="yaw_run_rate" value="0.75" />
    <remap from="velocity_cmd" to="cmd_vel"/>
  </node>
</launch>
